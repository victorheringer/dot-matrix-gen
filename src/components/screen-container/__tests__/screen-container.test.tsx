import { render } from "@testing-library/react";
import { ScreenContainer } from "../index";

describe("ScreenContainer", () => {
  it("should render correctly", () => {
    const tree = render(<ScreenContainer />).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
