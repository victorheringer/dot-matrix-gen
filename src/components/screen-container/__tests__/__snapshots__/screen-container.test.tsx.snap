// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenContainer should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ JThRt"
  />
</DocumentFragment>
`;
